app/controllers/api/v1/conversations_controller.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    # Controller class that is responsible for handling conversation requests.
+    class ConversationsController < ApplicationController
+      include DeviseTokenAuth::Concerns::SetUserByToken
+      include Pundit
+
+      before_action :authenticate_user!
+      before_action :find_conversation, only: %i[show destroy update]
+      before_action :find_conversation_id, only: %i[add_user delete_user]
+      before_action :authorize_conversation!
+      before_action :verify_authorized
```

Comment:
I was wrong here
https://github.com/varvet/pundit#ensuring-policies-and-scopes-are-used
But I'm not sure how it would help us to verify authorization as soon as we have `before_action :authorize_conversation!`

Lets remove line 14